### PR TITLE
Kevinf java updates

### PR DESCRIPTION
**Pull request by kfansler**
Saved OAuth bits to Google docs. Simplified this to Setting the access
token, List Sheets, and then Get Sheets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +24,7 @@ Add the SDK as a dependency in your project.
 </dependency>
 ```
 
-### 2. Download Jar File
+### Install By Downloading the Jar File
```
Comment (seweil) [inline README.md]: Please add a link to the sonatype library page. Best I can do is: https://search.maven.org/#search%7Cga%7C1%7Ca%3A%22smartsheet-sdk-java%22
Direct download of a JAR feels dangerous and abrupt, so we should provide an alternative as well.

File: README.md [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
+
+The SDK supports Java version 1.6 or later.
+
 ## Installation
 There are three different ways to install the SDK. Select the one that fits your environment best:
 
-1. [Use Maven](#1-maven)
-2. [Download Jar File](#2-download-jar-file)
-3. [Compile From Source](#3-compile-from-source)
+* [Install By Using Maven](install-by-using-maven)
```
Comment (seweil) [inline README.md]: These hyperlinks are broken in the preview. Please make sure they work when published.

File: README.md [hunk 3]
```diff
@@ -57,91 +67,40 @@ import com.smartsheet.api.oauth.*;
...
-
-    // On this page pass in the full URL of the page to create an authorizationResult object
-    AuthorizationResult authResult = oauth.extractAuthorizationResult(authorizationResponseURL);
-
-    // Get the token from the authorization result
-    Token token = oauth.obtainNewToken(authResult);
-
-    // Save the token or use it.
-}
+// Set the Access Token
```
Comment (seweil) [inline README.md]: The improved sample no longer requires either of these:
import com.smartsheet.api.models.enums.SourceInclusion;
import com.smartsheet.api.models.enums.ColumnType;

File: README.md [hunk 4]
```diff
@@ -57,91 +67,40 @@ import com.smartsheet.api.oauth.*;
...
+Token token = new Token();
```
Comment (seweil) [inline README.md]: You are missing creation of the client

File: README.md [hunk 5]
```diff
@@ -57,91 +67,40 @@ import com.smartsheet.api.oauth.*;
...
+
+// Sample 1: Omit 'include' parameter and pagination parameters
+PagedResult<Sheet> sheets = smartsheet.sheetResources().listSheets(
+    null,           // EnumSet<SourceInclusion> includes
+    null,           // PaginationParameters
+    null            // Date modifiedSince
+);
+
+// Sample 2: Specify pagination parameter 'includeAll' 
+PaginationParameters parameters = new PaginationParameters.PaginationParametersBuilder().setIncludeAll(true).build();
```
Comment (seweil) [inline README.md]: I wouldn't show pagination here. Not required and covered in the docs.

Please copy the C# example and use the first sheetId returned as the sheet to get. (Leave a commented line to allow the user to override.)

File: README.md (hunk 2 above)
Comment (kfansler) [inline README.md]: I confused (#1) with "number 1" instead of (#) to go to an internal link followed by 1.

**Pull request merged**